### PR TITLE
Lvnodcdxi 11

### PR DESCRIPTION
**Pull request by Kindest13**
Created simple react routing with client folder structure & added front modules

### CODE REVIEW

File: src/client/App.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import Login from './containers/Login/Login';
+
+class App extends Component {
+
+  render() {
+    let routes = (
+      <Switch>
+        <Route path="/" exact component={Main} />
+        <Route path="/auth" component={Login} />
+        <Route path="/regist" component={Registration} />
```
Comment (tonypntsrn) [inline src/client/App.js]: Better to not short your routes, and leave full name, for eg. “registration” or etc.

File: src/client/containers/Main/Main.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React, {Component} from 'react';
+import { Link } from 'react-router-dom';
+
+class Main extends Component {
+    render() {
+        return (
+            <div>
```
Comment (tonypntsrn) [inline src/client/containers/Main/Main.js]: You could use React.fragment here

File: src/server/app.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const express = require('express');
```
Comment (tonypntsrn) [inline src/server/app.js:1]: If you are using es6 for all app, better will be  to destruct like this import { express } from “express”

**Comment by Kindest13** [comment]
fixed&updated, review

**Review by tonypntsrn — APPROVED** [review]

**Pull request closed without merging**